src/pages/SongPage.tsx

```diff
@@ -34,7 +35,9 @@ const SongPage: React.FC = () => {
 
       <IonContent>
         {/* TODO: Add error handling in case of non number song Id */}
+        {PrevButton(+songId)}
```

Comment:
> - I want this to be a web only change (mobile should swipe for page change imo)

This can be done by making use of the `platformUtils` to check if it is in a browser.
https://github.com/Church-Life-Apps/Songs/blob/master/src/utils/PlatformUtils.tsx#L29-L31

Code would probably look like `{isBrowser() && PrevButton(+songId)}`, and same with line 40.